### ISSUE TITLE
Change the ban UI

### ISSUE THREAD

**Issue by thePlebDev** [thePlebDev/Clicker#498]
# Proposed change
- There should be no duration on the ban UI, it should just be permanent 

# Why is this important
- This is what the user expects

# Additional context
- n/a

**Comment by thePlebDev** [comment]
- should also add this to the Timeout dialog for a week off:
```
Column {
       RadioButton(
       colors =  RadioButtonDefaults.colors( selectedColor=secondary, unselectedColor = onPrimary),
       selected = banDuration == 604800,
       onClick = { changeBanDuration(604800) }
        )
        Text(stringResource(R.string.one_week),color = onPrimary)
}

```

**Comment by thePlebDev** [comment]
- closed with #499
